packages/contracts/contracts/optimistic-ethereum/libraries/utils/Lib_BytesUtils.sol

```diff
@@ -93,7 +93,7 @@ library Lib_BytesUtils {
             bytes memory
         )
     {
-        if (_bytes.length - _start == 0) {
+        if (_start >= _bytes.length) {
```

Comment:
Was this failing the tests before? Could we run in a bug in our currently deployed networks?